CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [29.0.1]
+### Added
+- Add utils to format token list tokens for aggregators and icon url. ([#810](https://github.com/MetaMask/controllers/pull/810))
+- Add SupportedTokenDetectionNetworks enum and util for token detection support. ([#811](https://github.com/MetaMask/controllers/pull/811))
+
+### Fixed
+- Fix bug introduced in v29.0.0 where serialized seedphrases passed to `createNewVaultAndRestore` were not correctly validated. ([#819](https://github.com/MetaMask/controllers/pull/819))
```

Comment:
Nit: This makes it sound like it was allowing invalid seeds, or something like that. Maybe clearer to say it was crashing.
```suggestion
- Fix bug introduced in v29.0.0 where `createNewVaultAndRestore` would crash with a validation failure. ([#819](https://github.com/MetaMask/controllers/pull/819))
```